packages/rlc-common/src/metadata/buildPackageFile.ts

```diff
@@ -57,6 +58,14 @@ export function buildPackageFile(
     packageInfo = buildAzureStandalonePackage(extendedConfig);
   }
 
+  if (
+    model.options?.moduleKind == "esm" &&
```

Comment:
why limit the moduleKind to esm? does that cjs would have no metadata?

Reply:
cjs will use the metadata from buildAzureMonorepoPackage  directly